docs/contributing/Git-mini-guide.md

```diff
@@ -0,0 +1,196 @@
+This is a mini-quick guide about using Git to share patches for wxWidgets.
+Better and extensive information can be found at
+https://git-scm.com/book/en/v2  and  https://git-scm.com/docs
+
+What is Git?
+------------
+Git is a distributed version control system.
+'Version control' is a system that records changes to a file or set of files
+over time so that you can recall specific versions later.
+'Distributed' means that every developer makes a full copy of the files in the
+repository of the project, including their full history of changes.
+
+What is GitHub?
+---------------
+It's a host of Git repositories. Its site https://github.com is a web interface
+for repositories. wxWidgets is just one of many projects that uses a repository
+that you can browse or download or collaborate with.
+
+1. Install git
+--------------
+Download from https://git-scm.com/downloads the version you like. It can be a
+terminal-based or a GUI one.
+In this guide I use the terminal, just to show the gist of the usage of git.
+
+Git needs to identify you by user/mail because your uploads will carry this info.
+   git config --global user.name "My name"
+   git config --global user.email my@domain.com
+Using '--global' will ask you just this time, no matter what project you work on.
+
+2. Download wxWidgets code... perhaps not yet
+---------------------------------------------
+If you just want the last snap of wx's code, don't plan to modify it, then you
+don't need to create a "divergence" from the "main-stream" of wx. You may
+simply download the repository, compile wx and continue with your business.
+But if you want to contribute, you need a "fork" and a "branch".
+
+3. Create a fork
+----------------
+This is where you tell GitHub you create a divergence to upload code to.
+First, sign in at https://github.com You may need to first create an account.
+I advise to use the same user/email as you did with Git configuration.
+
+A user & password is not enough to upload code. You need other authentication mode.
+You may use SSH keys. You may use PAT.
+See the Git-book and
+https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls
+
+
+Now you've signed in, create a fork.
+  Browse to https://github.com/wxWidgets/wxWidgets
+  About the top right corner find and use the "Fork" button. Select "Create a new fork"
+The new fork will be created as github.com/<your_github_name>/wxWidgets.git
+
+This fork is your fork. You don't need another fork to submit changes for
+another contribution to wxWidgets.
+
+4. Download wxWidgets code, now yes
+-----------------------------------
+Create the directory 'wxFork' (or the name you like).
+
+Option a)
+  git clone --recurse-submodules https://github.com/wxWidgets/wxWidgets.git wxFork
+Option b)
+  git clone --recurse-submodules https://github.com/<your_github_name>/wxWidgets.git wxFork

```

Comment:
Here you need "option b" only.
You will not get a write access to the main wxWidgets repository anyway.

Just clone your fork.

I believe even Vadim work out of his own fork. ;-)